src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,11 +5,39 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private int bladeWidth;
```

Comment:
Don't forget about access modifiers for your custom fields in `Bulldozer` class. It's a good practice to encapsulate fields using private access modifier and provide public getters and setters for accessing them.